vendor/engines/split_accounts/spec/controllers/facility_accounts_controller_spec.rb

```diff
@@ -66,8 +39,61 @@
     end
   end
 
-  describe "as a facility administrator" do
-    let(:user) { FactoryGirl.create(:user, :facility_administrator, facility: facility) }
+  shared_examples "allows suspending" do
+    let(:split_account) { FactoryGirl.create(:split_account) }
+
+    describe "show" do
+      before { get :show, facility_id: facility.url_name, id: split_account.id }
+
+      it "includes the suspend button" do
+        expect(response.body).to include("Suspend")
+      end
+    end
+
+    describe "suspending" do
+      it "suspends the account" do
+        expect { get :suspend, facility_id: facility.url_name, id: split_account.id }
+          .to change { split_account.reload.suspended_at }
+      end
+    end
+  end
+
+  describe "as an admin" do
+    let(:user) { FactoryGirl.create(:user, :administrator) }
+
+    include_examples "allows editing"
+    include_examples "allows suspending"
+
+    describe "default new" do
+      before { get :new, facility_id: facility.url_name, owner_user_id: user.id }
+
+      it "sees split account option" do
+        expect(response.code).to eq("200")
+        expect(response.body).to include("Chart String")
+        expect(response.body).to include("Split Account")
```

Comment:
I played with it, and it got super awkward for the examples with `not_to`, and I'd rather be a little verbose but remain consistent within the file. Using `:aggregate_failures` should accomplish the same upside, though, so I went ahead and added that.